roles/nvidia-gpu-operator/tasks/main.yml

```diff
@@ -1,4 +1,15 @@
 ---
+- name: Check for DGX packages
```

Comment:
This is not going to work.

I was testing this on a single DGX-station, but in a real deployment scenario the admin node will not have a /etc/dgx-release even if all the workers are DGX systems.

We need to come up with a better method to determine if there are any DGX OS systems within the cluster.

Reply:
we could still run a task like this on every node, then tag them as 'dgx' in k8s

Reply:
I bet the GPU Operator is already adding a tag for that, let me check.